.gitignore

```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /tests/phpunit.xml
 /vendor
+.idea
```

Comment:
Only project specifics should go in this gitignore, you can ignore stuff related to your personal setup with a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore